consensus/entry_point_selector/entry_point_selector.c

```diff
@@ -26,27 +26,31 @@ retcode_t iota_consensus_entry_point_selector_init(
   return RC_OK;
 }
 
-retcode_t iota_consensus_get_entry_point(entry_point_selector_t *const eps,
-                                         size_t const depth,
-                                         trit_array_p const ep) {
+retcode_t iota_consensus_entry_point_selector_get_entry_point(
+    entry_point_selector_t *const eps, size_t const depth,
+    trit_array_p const ep) {
   retcode_t ret = RC_OK;
-  size_t milestone_index = MAX(eps->mt->latest_milestone_index - depth - 1, 0);
+  uint64_t milestone_index =
+      MAX(eps->mt->latest_solid_subtangle_milestone_index - depth - 1, 0);
```

Comment:
~I believe you're missing a cast to `int64_t` here?~ You're not, but we probably shouldn't use `size_t` for `latest_solid_subtangle_milestone_index`?

Reply:
Ah it's not on the `get_node_info` struct - unrelated.
So yeah, you definitely want to cast here?

Reply:
could use `ULLONG_MAX`